platform/ios/src/MGLMapView.h

```diff
@@ -1130,7 +1130,7 @@ MGL_EXPORT
  `UIApplicationWillTerminateNotification`; you may receive a `nil` return value
  depending on the order of notification delivery.
  */
-- (MGLMapCamera *)cameraThatFitsCoordinateBounds:(MGLCoordinateBounds)bounds edgePadding:(UIEdgeInsets)insets;
+- (nullable MGLMapCamera *)cameraThatFitsCoordinateBounds:(MGLCoordinateBounds)bounds edgePadding:(UIEdgeInsets)insets;
```

Comment:
This is a backwards-incompatible change. This method only returns `nil` in the case where the underlying `mbgl::Map` has gone away. That case should only happen at teardown, so we’d be entirely justified in returning a newly initialized MGLMapCamera with zeroes for all its parameters. (×4)